client/src/component/Sign-Up-Page.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+
+class SignUpHacker extends Component {
```

Comment:
```suggestion
class SignUpHacker extends Component {
```

Let's make a centralized `SignUpComponent` that will be the basis for all signups and then we fill more data based on the the previously filled in textfields similar to Typeform, but dynamic across all types of users.